website/content/docs/configuration/server.mdx

```diff
@@ -43,11 +43,13 @@ server {
   `1` does not provide any fault tolerance and is not recommended for production
   use cases.
 
-- `data_dir` `(string: "[data_dir]/server")` - Specifies the directory to use
-  for server-specific data, including the replicated log. By default, this is
-  the top-level [data_dir](/nomad/docs/configuration#data_dir) suffixed with "server",
-  like `"/opt/nomad/server"`. The top-level option must be set, even when
-  setting this value. This must be an absolute path.
+- `data_dir` `(string: "")` - Specifies the directory to use for server-specific
+  data, including the replicated log. When this parameter is empty, Nomad will
+  generate the path using the [top-level data_dir][top_level_data_dir] suffixed
+  with "server", like `"/opt/nomad/server"`. The
+  [top-level data_dir][top_level_data_dir] must be set, even when setting this
+  value. This must be an absolute path. Nomad will create the directory on the
+  host, if it does not exist when the agent process starts.
```

Comment:
Yeah good shout; this is the only one that does it, because of the Raft setup. I'll clarify the others must exist on the host, otherwise startup will fail.

Reply:
Added clarification within https://github.com/hashicorp/nomad/pull/20268/commits/16fb731ea4bc9336677559e08af711eb501a0242 which was learning for me.